stdlib/2and3/turtle.pyi

```diff
@@ -0,0 +1,487 @@
+            outline: int = ..., tilt: float = ...) -> None: ...
+    width = pensize
+    up = penup
+    pu = penup
+    pd = pendown
+    down = pendown
+    st = showturtle
+    ht = hideturtle
+
+_T = TypeVar('_T')
+
+class RawTurtle(TPen, TNavigator):
+    def __init__(self, canvas: Optional[Canvas] = ..., shape: str = ..., undobuffersize: int = ..., visible: bool = ...) -> None: ...
+    def reset(self) -> None: ...
+    def setundobuffer(self, size: Optional[int]) -> None: ...
+    def undobufferentries(self) -> int: ...
+    def clear(self) -> None: ...
+    def clone(self: _T) -> _T: ...
+    @overload
+    def shape(self) -> str: ...
+    @overload
+    def shape(self, name: str) -> None: ...
+    # Unsafely overlaps when no arguments are provided
+    @overload
+    def shapesize(self) -> Tuple[float, float, float]: ...  # type: ignore
+    @overload
+    def shapesize(self, stretch_wid: Optional[float] = ..., stretch_len: Optional[float] = ..., outline: Optional[float] = ...) -> None: ...
+    if sys.version_info >= (3,):
+        @overload
+        def shearfactor(self) -> float: ...
+        @overload
+        def shearfactor(self, shear: float) -> None: ...
+        # Unsafely overlaps when no arguments are provided
+        @overload
+        def shapetransform(self) -> Tuple[float, float, float, float]: ...  # type: ignore
+        @overload
+        def shapetransform(self, t11: Optional[float] = ..., t12: Optional[float] = ..., t21: Optional[float] = ..., t22: Optional[float] = ...) -> None: ...
+        def get_shapepoly(self) -> Optional[_PolygonCoords]: ...
+    def settiltangle(self, angle: float) -> None: ...
+    @overload
+    def tiltangle(self) -> float: ...
+    @overload
+    def tiltangle(self, angle: float) -> None: ...
+    def tilt(self, angle: float) -> None: ...
+    def stamp(self) -> int: ...
+    def clearstamp(self, stampid: int) -> None: ...
+    def clearstamps(self, n: Optional[int] = ...) -> None: ...
+    def filling(self) -> bool: ...
+    def begin_fill(self) -> None: ...
+    def end_fill(self) -> None: ...
+    def dot(self, size: Optional[int], *color: _Color) -> None: ...
+    def write(self, arg: object, move: bool = ..., align: str = ..., font: Tuple[str, int, str] = ...) -> None: ...
+    def begin_poly(self) -> None: ...
+    def end_poly(self) -> None: ...
+    def get_poly(self) -> _PolygonCoords: ...
+    def getscreen(self) -> TurtleScreen: ...
+    def getturtle(self) -> Turtle: ...
+    getpen = getturtle
+    def onclick(self, fun: Callable[[float, float], Any], btn: int = ..., add: Optional[Any] = ...) -> None: ...
+    def onrelease(self, fun: Callable[[float, float], Any], btn: int = ..., add: Optional[Any] = ...) -> None: ...
+    def ondrag(self, fun: Callable[[float, float], Any], btn: int = ..., add: Optional[Any] = ...) -> None: ...
+    def undo(self) -> None: ...
+    turtlesize = shapesize
+
+class _Screen(TurtleScreen):
```

Comment:
We should add `__init__()` as it doesn't accept any arguments, while the superclass's init does.